### DISCUSSION TITLE
[ZK Stack] - zk stack init error - Invalid mount config for type "bind"

### DISCUSSION THREAD

**Discussion by jkjx** [zkSync-Community-Hub/zksync-developers#470] [ZK Stack] [answered]
### Module Affected

- [ ] System contracts (era-system-contracts)
- [ ] Smart contracts (era-contracts)
- [ ] zkEVM (era-zk_evm)
- [X] Server (zksync-era)
- [ ] Prover (era-boojum)
- [ ] Fee withdrawer (era-fee-withdrawer)
- [ ] Block Explorer (clock-explorer)
- [ ] Portal - Wallet + Bridge (dapp-portal)
- [ ] Other (please specify below)

### Rust Version

1.27.0

### Operating System

- [X] Ubuntu
- [X] MacOS
- [ ] Windows
- [ ] Other (Please specify below)

### Issue Description

I am following the guide on https://docs.zksync.io/zk-stack/running-a-hyperchain/locally.html#deploying-locally.
When i run >>zk stack init, this error shows up:
Error response from daemon: invalid mount config for type "bind": bind source path does not exist: /host_mnt/Users/user/zksync-era/volumes/postgresI have tried this on both mac and ubuntu and failed.
the zksync-era/volumes folder is not present when running this command.
I have seen others post this error on discord but have not gotten any responses.
What can i do?

### Expected Behavior

Expected to have the hyperchain deployed.

### Current Behavior

Error message shows up. Shows up also when I am running the demo. >>zk stack demo. Same error.

### Prior Research

- [X] Yes, I searched for similar issues and did not find any matching ones.

**Comment by bxpana** [comment]
Will see if I can find out more for you on this.

**Comment by githubdoramon** [comment, marked as answer]
@jkjx we are working on a fully revamped zk cli that will fix all of these issues. Please hand in tight for a week or 2

**Comment by Vedantjn** [comment]
@jkjx how did you resolved it?
